internal/db/schema/migrations/postgres/16/03_host.up.sql

```diff
@@ -0,0 +1,63 @@
+begin;
+
+  -- We are adding the name to the base host catalog type. This allows the db
+  -- to ensure that catalog names are unique in a scope across all subtypes.
+  alter table host_catalog
+    add column name wt_name;
+
+  alter table host_catalog
+    add constraint host_catalog_scope_id_name_uq
+      unique (scope_id, name);
+
+  -- Now that we've added the name column to the base type, we copy
+  -- the name from the static host catalog table into the base table.
+  update host_catalog
+  set name = st.name
+  from
+    static_host_catalog st
+  where
+    host_catalog.public_id = st.public_id and
+    st.name is not null;
+
+  -- Replace the insert_host_catalog_subtype function defined in 0/20_host.up.sql
+  -- to include the name.
+  -- insert_host_catalog_subtype() is a before insert trigger
+  -- function for subtypes of host_catalog
+  create or replace function
```

Comment:
Should there be a `create trigger` statement for this function?

Reply:
This is in the 04_plugin_host.up.sql file.